simpledotorg/simple-server
# Add pending facility graphs to diabetes dashboard

**Story card:** 
- [sc-8295](https://app.shortcut.com/simpledotorg/story/8295/in-dm-reports-for-a-facility-add-patient-registration-and-follow-up-by-users-table)
-  [sc-8296](https://app.shortcut.com/simpledotorg/story/8296/in-diabetes-reports-for-a-facility-add-health-care-worker-activity-section)
- [sc-8297](https://app.shortcut.com/simpledotorg/story/8297/in-diabetes-reports-for-a-facility-add-the-recent-blood-sugar-measurement-tables)

## Because

These are pending graphs on the diabetes dashboard which were not part of the original scope

## This addresses

Adds cards to facility diabetes dashboard for
- Registrations and follow ups by user
- Healthcare worker activity
- Recent blood sugar measurements at a facility

It also adds a card to the admin/user page for recent blood sugar measurements by a user. 

## Test instructions

- Checkout this branch
- Turn on `:diabetes_management_enabled` flipper flag
- Look at the diabetes reports for a facility. The facility should have `enable_diabetes_management` set to true
- You should see the following graphs
  - Registrations and follow ups by user
  - Healthcare worker activity
  - Recent blood sugar measurements at a facility
 - Click on a username. It should take you to an admin/user page. Here you should be able to the recent blood sugar measurements by a user.
